src/utils/utils.js

```diff
@@ -1,10 +1,14 @@
 import axios from 'axios';
 
-const getArticles = query => {
+const getArticles = (query, sort_by, order) => {
+  // console.log(sort_by);
+
   return axios
     .get('https://nc-news-kx4n.onrender.com/api/articles', {
       params: {
         topic: query,
+        sort_by: sort_by,
+        order: order,
```

Comment:
Love your use of the params functionality here.
You should consider also using the baseURL functionality in this file as well so if you ever need to change the API web address you would only need to in one place https://notes.northcoders.com/courses/js-front-end/axios#axios-instance